lib/wicked/wlan.pm

```diff
@@ -69,25 +103,44 @@ sub netns_output {
     return script_output($cmd, @args);
 }
 
+sub netns_run {
+    my ($self, $cmd, @args) = @_;
+    $cmd = 'ip netns exec ' . $self->netns_name . ' ' . $cmd;
+    my $ret = script_run($cmd, @args);
+    die("Timeout on script_run($cmd)") unless defined($ret);
+    return $ret;
+}
+
+sub DHCP_pidfile {
```

Comment:
would you mind to make them lowcase? or is there any reason?

Reply:
no, I will lower it